### PR TITLE
Feature/redesign definitions panel

### PR DESCRIPTION
**Pull request by Zorin95670**
- Create `LibraryList.vue` and `LibraryItem.vue`
- Inside Components Drawer (`ModelizerModelView`)
  - Display list of libraries
  - After selecting a library, display grid of its components
  - Display grid of components with flex instead of using `q-table`

### CODE REVIEW

File: src/components/list/LibraryList.vue [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        @click="onItemClick(drawerItem.id)"
+      />
+    </template>
+    <q-item
+      v-if="selectedItemId && !isEmptyList"
+      style="overflow-y: auto !important;"
+      class="q-pa-none q-mb-sm"
+    >
+      <component-definition-grid
+        :definitions="drawerItems.find(({ id }) => selectedItemId === id)?.definitions"
```
Comment (k-y0u) [inline src/components/list/LibraryList.vue:37]: why not create a computed ?

File: src/composables/Project.js [hunk 2]
```diff
@@ -771,3 +771,15 @@ export async function renameProject(projectId, newProjectName) {
...
+
+/**
+ * Indicate if searched text is found.
+ * @param {String} filter - Searched text.
+ * @param {String} value - Value to match with.
+ * @return {Boolean} true if it is matching otherwise false.
+ */
+export function isMatching(filter, value) {
+  return !filter || !filter.trim() || filter.toLowerCase().split(' ')
+    .filter((searchedText) => searchedText)
```
Comment (k-y0u) [inline src/composables/Project.js:783]: what's the point of this filter ?

File: src/components/item/LibraryItem.vue [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<template>
+  <q-item
+    v-ripple
+    clickable
+    :class="isSelected ? 'border-bottom selected' : ''"
+    :data-cy="`component-defnitions-item_${item.title}`"
```
Comment (Zorin95670) [inline src/components/item/LibraryItem.vue]: typo: `defnitions`

File: cypress/e2e/ModelizerPage/ModelView/plugin/filter.plugin.feature [hunk 4]
```diff
@@ -1,4 +1,4 @@
-Feature: Test modelizer model view: plugin initialization
+Feature: Test modelizer model view: plugin filter
```
Comment (k-y0u) [inline cypress/e2e/ModelizerPage/ModelView/plugin/filter.plugin.feature:1]: modelizer draw view

File: cypress/e2e/ModelizerPage/ModelView/template/display.component.feature [hunk 5]
```diff
@@ -0,0 +1,57 @@
+Feature: Test modelizer model view: add template component
```
Comment (k-y0u) [inline cypress/e2e/ModelizerPage/ModelView/template/display.component.feature:1]: modelizer draw view

File: changelog.md [hunk 6]
```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 * Update e2e test plugin to terrator-plugin.
+* New display of libraries components inside ComponentDefinitionsDrawer..
```
Comment (Zorin95670) [inline changelog.md:25]: `..` -> `.`

File: cypress/e2e/ModelizerPage/ModelView/template/filter.template.feature [hunk 7]
```diff
@@ -0,0 +1,96 @@
+Feature: Test modelizer model view: template filter
```
Comment (k-y0u) [inline cypress/e2e/ModelizerPage/ModelView/template/filter.template.feature:1]: modelizer draw view

File: cypress/e2e/ModelizerPage/ModelView/template/init.template.feature [hunk 8]
```diff
@@ -0,0 +1,35 @@
+Feature: Test modelizer model view: template initialization
```
Comment (k-y0u) [inline cypress/e2e/ModelizerPage/ModelView/template/init.template.feature:1]: modelizer draw view

File: changelog.md (hunk 6 above)
Comment (k-y0u) [inline changelog.md:25]: New display for ComponentDefinitionsDrawer: library list & component grid separate.

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 6 above)
Comment (Zorin95670) [reply changelog.md:25]: `Rework on display of the components list.` May be?

File: src/components/item/LibraryItem.vue (hunk 3 above)
Comment (justinehell) [reply src/components/item/LibraryItem.vue]: done

File: changelog.md (hunk 6 above)
Comment (justinehell) [reply changelog.md:25]: changed

Comment (justinehell) [reply changelog.md:25]: changed

File: cypress/e2e/ModelizerPage/ModelView/template/display.component.feature (hunk 5 above)
Comment (justinehell) [reply cypress/e2e/ModelizerPage/ModelView/template/display.component.feature:1]: done

File: cypress/e2e/ModelizerPage/ModelView/template/filter.template.feature (hunk 7 above)
Comment (justinehell) [reply cypress/e2e/ModelizerPage/ModelView/template/filter.template.feature:1]: done

File: cypress/e2e/ModelizerPage/ModelView/template/init.template.feature (hunk 8 above)
Comment (justinehell) [reply cypress/e2e/ModelizerPage/ModelView/template/init.template.feature:1]: done

File: src/components/list/LibraryList.vue (hunk 1 above)
Comment (justinehell) [reply src/components/list/LibraryList.vue:37]: done

**Review by k-y0u — APPROVED** [review]

**Pull request merged**